app/src/main/java/net/gsantner/markor/activity/DocumentShareIntoFragment.java

```diff
@@ -30,7 +30,7 @@
 import net.gsantner.markor.R;
 import net.gsantner.markor.format.FormatRegistry;
 import net.gsantner.markor.format.plaintext.PlaintextSyntaxHighlighter;
-import net.gsantner.markor.format.todotxt.TodoTxtParser;
```

Comment:
The TodoTxtParser / TodoTxtTask class is used to both parse a task as well as hold an instance of a parsed task. IMO naming it TodoTxtTask is better.

Reply:
My opinion: A todo.txt task = 1 line of text, or being model of 1 line.

What this however does is parse, convert, transform stuff so I thought Parser to be fitting.

But at the end, I don't care. If you want to rename it and feel it's bettery it is OK^^.

Reply:
> My opinion: A todo.txt task = 1 line of text, or being model of 1 line.

This is exactly what TodoTxtTask is. It's just that the parsing logic is encapsulated into the same class.